knex/knex-tests.ts

```diff
@@ -1,5 +1,6 @@
 "use strict";
 import * as Knex from 'knex';
+import * as Promise from 'bluebird';
```

Comment:
Sorry about this - can you change this to `import x = require("foo")` here as well please?